lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -151,7 +151,14 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
             }
 
             if (this.isPathVariableType(DestinationFolderPathType.FOLDER_TYPE)) {
-                rootNodeId = this.field.params.fileSource.destinationFolderPath.value;
+                try {
+                    const isNodeAvailable = await this.contentNodeSelectorService.isNodeAvailable(this.field.params.fileSource.destinationFolderPath.value);
+                    if (isNodeAvailable) {
+                        rootNodeId = this.field.params.fileSource.destinationFolderPath.value;
```

Comment:
do we need to return the available node in this case? if yep shortcut the method by calling the return... do not let the developer to go though additional useless steps

Reply:
if the node is available means that is not null so u don;t need to reach `  if (!rootNodeId) {
            try {
                const nodeId = await this.getNodeIdBasedOnPath(destinationFolderPath);
                rootNodeId = nodeId ? nodeId : destinationFolderPath.alias;
            } catch (error) {
                this.logService.error(error);
            }
        }`

Reply:
Updated